### ISSUE TITLE
LMDE/Mint 20 proposed updates

### ISSUE THREAD

**Issue by ad5xj** [teejee2008/timeshift#565]
The proposed Timeshift changes for LMDE 4 and Mint 20 are needed. But things left out are the choice of when a backup is done. An enhancement to the settings schedule is sorely needed such as:

    for Monthly - a choice of what day
    for weekly - a choice of which day of the week.
    for Daily - what time of day to start the backups
    for hourly - what hour of the day.

Right now none of these items are selectable and they need to be in order for Timeshift to be really useful.

**Comment by LDLanagan** [comment]
Agreed. My computer is slow enough that Timeshift really doesn't run in the background. It slows everything down. Those ideas are golden.

**Comment by camilova** [comment]
Its would be very helpfull, so I can set daily at end of day, and get a snapshot everyday with all work arround the day.
